### PR TITLE
Center on mouse during zoom

### PR DESCRIPTION
**Pull request by ajrgrubbs**
Occurs only during double-click to zoom, obviously.

### CODE REVIEW

File: vee/static/treeplayer.js [hunk 1]
```diff
@@ -296,23 +296,28 @@ class TreePlayer {
...
     tv.scale = newScale;
     tv.x = ((tv.x - w) * d) + w;
     tv.y = ((tv.y - h) * d) + h;
     this.updateTransforms();
   }
 
-  zoomIn() {
-    this.changeScale(0.1);
+  zoomIn(coords) {
+    var x, y;
```
Comment (csavage1994) [inline vee/static/treeplayer.js]: I know Dan doesn't like this due to the possibility of creating global variable on accident if someone needs to edit this list of variables and they miss a comma

Comment (ajrgrubbs) [inline vee/static/treeplayer.js]: Good to know

**Comment by csavage1994** [comment]
👍

**Comment by vforgione** [comment]
👍

**Pull request merged**